umap/static/umap/js/bindModules.mjs

```diff
@@ -0,0 +1,3 @@
+import URLs from "./modules/urls.mjs"
+
+window.URLs = URLs;
```

Comment:
We have to bind the loaded modules to something in order to access it on the other side.

Reply:
missing newline? ⏎

Not use about `.mjs` vs `.js` but a reading is here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#aside_%E2%80%94_.mjs_versus_.js